CRM/Utils/REST.php

```diff
@@ -546,6 +546,14 @@ public static function ajaxJson() {
       );
       CRM_Utils_JSON::output($error);
     }
+    // FIXME: Shouldn't the X-Forwarded-Proto check be part of CRM_Utils_System::isSSL()?
```

Comment:
@ineffyble isn't most of this covered in the CRM_Utils_System::redirectToSSL and does it need the CMS to be bootstraped for the redirect?